### DISCUSSION TITLE
Bit of a rewrite, looking for feedback.

### DISCUSSION THREAD

**Discussion by honusz** [MarByteBeep/StageLinq#25] [Show and tell]
Hey everyone,
I've been tooling around a bit on my fork of StageLinq, I'd love to get some feedback / tests with different devices & configurations.

https://github.com/honusz/StageLinq

The objective is to move this project from it's experimental origins, to a complete library. I'm not necessarily saying this fork _should_ be how we do this, just one proposed direction.

Some highlights of the work I've done:

* Updated dependancies.
* Emphasis on Stability: One of the goals has focusing on edge cases, attempting to make the library rock-solid, no matter whats thrown at it. It handles user behaviour like:
       * USB Sticks on separate devices with the same name.
       * Disconnecting USB and moving it to another player.
       * Multiple Sources on the same Unit.
       * Player disconnects or reboots.
* BeatInfo service implementation.
* Experimental framework for TimeSynchronization & Broadcast services.

There's some info in the ReadMe, and I've started putting in the bones of TypeDoc documentation.
Anyway, I'd love to hear what everyone thinks. And please, let me know if anything doesn't work as expected.

Thanks!

**Comment by jaygarcia** [comment]
I am going to locate some time this week to exercise this fork. 😁
